lms/templates/courseware/progress.html

```diff
@@ -59,10 +59,15 @@ <h1 class="progress-certificates-title">${_("Course Progress for Student '{usern
             <div class="auto-cert-message" id="course-success">
                 <div class="has-actions">
                 %if passed:
+                    <input type="hidden" id="post_cert_url" value="${reverse('generate_user_cert', args=[unicode(course.id)])}" />
```

Comment:
We were using the reverse at two places thats why i did this way. But code is updated now.